MAUI/NavigationDrawer/Side-Pane-Content.md

```diff
@@ -0,0 +1,363 @@
+    </navigationdrawer:SfNavigationDrawer.ContentView>
+</navigationdrawer:SfNavigationDrawer>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.NavigationDrawer;
+
+namespace NavigationSample;
+
+public partial class NavigationDrawerView : ContentPage
+{
+	public NavigationDrawerView()
+	{
+		InitializeComponent();
+		navigationDrawer.DrawerSettings.DrawerWidth = 250;
+	}
+
+    void hamburgerButton_Clicked(object sender, EventArgs e)
+    {
+        navigationDrawer.ToggleDrawer();
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Footer]()
+
+## Footer Height
+
+The height of the drawer footer content can be adjusted using the [`DrawerFooterHeight`]() property.
+
+N> [`DrawerFooterView`]() will be disabled by setting the [`DrawerFooterHeight`]() to zero 
+
+{% tabs %}
+
+{% highlight xaml %} 
+
+<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+    <navigationdrawer:SfNavigationDrawer.DrawerSettings>
+        <navigationdrawer:DrawerSettings DrawerFooterHeight="150">
+            <navigationdrawer:DrawerSettings.DrawerFooterView>
+                <Grid BackgroundColor="#1aa1d6">
+                    <VerticalStackLayout VerticalOptions="Center"
+                                         HorizontalOptions="Center">
+                        <Label Text="Footer View"/>
+                    </VerticalStackLayout>
+                </Grid>
+            </navigationdrawer:DrawerSettings.DrawerFooterView>
+        </navigationdrawer:DrawerSettings>
+    </navigationdrawer:SfNavigationDrawer.DrawerSettings>
+    <navigationdrawer:SfNavigationDrawer.ContentView>
+        <VerticalStackLayout>
+            <HorizontalStackLayout BackgroundColor="#1aa1d6" 
+                  HeightRequest="50" 
+                  VerticalOptions="Start"
+                  Padding="5,0,0,0">
+                <ImageButton x:Name="hamburgerButton"
+                             HeightRequest="50"
+                             WidthRequest="50"
+                             HorizontalOptions="Start"
+                             Source="hamburgericon.png"
+                             BackgroundColor="#1aa1d6"
+                             Clicked="hamburgerButton_Clicked"/>
+            </HorizontalStackLayout>
+        </VerticalStackLayout>
+    </navigationdrawer:SfNavigationDrawer.ContentView>
+</navigationdrawer:SfNavigationDrawer>
+	
+{% endhighlight %}
+
+{% highlight c# %}
+        
+using Syncfusion.Maui.NavigationDrawer;
+
+namespace NavigationSample;
+
+public partial class NavigationDrawerView : ContentPage
+{
+	public NavigationDrawerView()
+	{
+		InitializeComponent();
+        navigationDrawer.DrawerSettings.DrawerWidth = 250;
+        navigationDrawer.DrawerSettings.DrawerFooterHeight = 150;
+	}
+
+    void hamburgerButton_Clicked(object sender, EventArgs e)
+    {
+        navigationDrawer.ToggleDrawer();
+    }
+}
+  
+{% endhighlight %}
+
+{% endtabs %}
+
+![Footer hight]()
+
+## Main Content
+
```

Comment:
The main content of the drawer is displayed between the header and footer content. It can be set using the `DrawerContentView` property. The content view occupies the remaining space left by the header and footer content.